app.js

```diff
@@ -1,11 +1,73 @@
 /* Imports */
+import { getBeanieBabies, getAstrology } from './fetch-utils.js';
 
 /* Get DOM Elements */
+const beanieBabiesEl = document.querySelector('.beanie-babies');
+const optionsEl = document.querySelector('.options');
+const selectEl = document.querySelector('select');
+const buttonEl = document.querySelector('button');
+const formEl = document.querySelector('form');
+
 
 /* State */
+let beanieBabiesData = [];
+let astrologyData = []; 
+
+const SUPABASE_URL = 'https://gxwgjhfyrlwiqakdeamc.supabase.co';
```

Comment:
you can remove the SUPABASE stuff and client here. you've already put that in fetch-utils. this isn't really doing anything in here.